engine/common/synchronization/engine.go

```diff
@@ -495,11 +489,10 @@ func (e *Engine) onSyncRequest(originID flow.Identifier, req *messages.SyncReque
 }
 
 // onSyncResponse processes a synchronization response.
-func (e *Engine) onSyncResponse(originID flow.Identifier, res *messages.SyncResponse) error {
+func (e *Engine) onSyncResponse(originID flow.Identifier, res *messages.SyncResponse) {
 
 	final := e.finalSnapshot().head
 	e.core.HandleHeight(final, res.Height)
-	return nil
```

Comment:
What's the logic for not inspecting errors and at least logging them here?

Reply:
I remove this line, because `HandleHeight` doesn't return error, and there isn't a case that would return a non-nil error. So I just removed this non-existent case